cardano-prelude-test/test/Test/Cardano/Prelude/GHC/Heap/SizeSpec.hs

```diff
@@ -103,6 +105,13 @@ mkBS :: [Word8] -> ByteString
 {-# NOINLINE mkBS #-}
 mkBS = BS.Strict.pack
 
+adjustBsSize :: NumWords -> NumWords
```

Comment:
can we get a comment on _why_ we have `-1` now?

Reply:
Done!